packages/waku/src/lib/builder/build.ts

```diff
@@ -1,46 +1,44 @@
-import { createHash } from 'node:crypto';
 import { Readable } from 'node:stream';
```

Comment:
IDK why I ran `prettier --write .` and order totally changed

Reply:
You are probably using a different version or a different config.

Reply:
Hope you figure it out...